onnxruntime/core/session/standalone_op_invoker.cc

```diff
@@ -143,6 +140,7 @@ class NodeRepo {
 
   std::mutex mutex_;
   StandAloneNodesMap node_map_;
+  StandAloneArgsMap arg_map_;
```

Comment:
would it be simpler to maintain a single map? e.g., from `OpKernel*` to `struct { NodePtr node; ArgPtrs args; }`
then we don't need to keep the maps in sync, do two lookups, etc.

Reply:
Done.